projects/event-api.md

```diff
@@ -0,0 +1,90 @@
+### Description
+
+The Client / RUM Sig is looking to finalize, complete and resolve the outstanding issues / PR's that exist around the Event API / SDK and the Semantic Conventions that defines what an "event" is.
+
+* Defined, identifying an `event`)
+* Created (API / Sdk)
+* will be created from Describe the goals, objectives, and requirements for the project. This include the motivations for starting the project now, as opposed to later.
+
+### Project Board
+
+Once approved by TC, a project should be managed using a GitHub project board. This project board should be pre-populated with issues that cover all known deliverables, organized by timeline milestones.
+
+A [Technical Committee](https://github.com/open-telemetry/community/blob/main/community-members.md#technical-committee) (TC) member associated with the project can create the board, along with a new project-specific GitHub label to automatically associate issues and PRs with the project. The project lead and all other relevant project members should have edit access to the board.
+
+Once created, please link to the project board here.
+
+Preemptively created a board with the relevant and related issues
+https://github.com/orgs/open-telemetry/projects/65
```

Comment:
Since there's already a project board can you delete the boiler plate above the link?